tests/jerry/es.next/proxy_define_own_property.js

```diff
@@ -56,33 +56,20 @@ var handler = {
 
 var proxy = new Proxy(target, handler);
 
-Object.defineProperty(proxy, "bar", desc);
-
-assert(proxy.bar === 1);
-
-/* TODO - remove this comment when [[GetOwnProperty]] is implemented
-proxy.bar = 2;
-assert(proxy.bar === 2);
-*/
-
-delete proxy.bar;
-assert(proxy.bar === undefined);
-
-/* TODO - remove this comment when [[GetOwnProperty]] is implemented
-Object.defineProperty(proxy, "name", {
-  get() {
-    return this._name;
-  },
-  set(value) {
-    this._name = value;
-  }
-});
+try {
+  Object.defineProperty(proxy, "bar", desc);
```

Comment:
After this the "bar" property should exist on the proxy/target, or not? We should add test/check for it.